The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data/pgdata ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data/pgdata -l logfile start


WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
waiting for server to start....2025-02-14 13:14:11.080 UTC [45] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 13:14:11.090 UTC [46] LOG:  database system was shut down at 2025-02-14 13:14:10 UTC
2025-02-14 13:14:11.092 UTC [45] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

waiting for server to shut down....2025-02-14 13:14:11.443 UTC [45] LOG:  received fast shutdown request
2025-02-14 13:14:11.443 UTC [45] LOG:  aborting any active transactions
2025-02-14 13:14:11.444 UTC [45] LOG:  background worker "logical replication launcher" (PID 52) exited with exit code 1
2025-02-14 13:14:11.444 UTC [47] LOG:  shutting down
2025-02-14 13:14:11.453 UTC [45] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2025-02-14 13:14:11.550 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-02-14 13:14:11.550 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-02-14 13:14:11.552 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 13:14:11.567 UTC [63] LOG:  database system was shut down at 2025-02-14 13:14:11 UTC
2025-02-14 13:14:11.570 UTC [1] LOG:  database system is ready to accept connections
2025-02-14 13:14:17.976 UTC [78] ERROR:  relation "_prisma_migrations" does not exist at character 126
2025-02-14 13:14:17.976 UTC [78] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
2025-02-14 13:14:18.360 UTC [78] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:14:18.360 UTC [78] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_36a0f026-901e-468f-ba2c-0046e38c82f2" WITH (FORCE)
2025-02-14 13:14:18.797 UTC [78] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:14:18.797 UTC [78] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_4bb20263-c313-44ed-bb8e-1d1b73dedd29" WITH (FORCE)
2025-02-14 13:14:19.323 UTC [78] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:14:19.323 UTC [78] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_06f89da9-c1eb-4bd0-9e2a-b2a9dd75208b" WITH (FORCE)
2025-02-14 13:14:19.828 UTC [78] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:14:19.828 UTC [78] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_ceb8d9f8-ffbd-4ad1-ba36-e0d5be1e1524" WITH (FORCE)
